### PR TITLE
Update filterable geo page

### PR DESCRIPTION
**Pull request by ghost**
The UI has been updated from **Filterable geography** to **Geography**. This PR contains changes made across several files to reflect the new UI name.

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx [hunk 1]
```diff
@@ -19,48 +20,48 @@ import browserFilterableGraphs from 'images/browser_screenshot-full_filterable-g
...
+If you work in IT or operations, this level of visual detail is especially useful for making business decisions about peering agreements and CDN usage. For example, if it's unclear which regions are experiencing higher page load times than normal, our **Geography** UI can help you track down the root of the problem.
 
 For some tips on how to make the most of the available data, see the [filter examples](#filter).
 
 ## Troubleshoot performance location [#view-data]
 
-Filterable geography functionality requires the [Pro or Pro+SPA agent type](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types). If you have not installed this agent type, you may be able to use the [legacy **Geography** UI page](#legacy-geo).
+The Geography UI functionality requires the [Pro or Pro+SPA agent type](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types). If you have not installed this agent type, you may be able to use the [legacy UI page](#legacy-geo).
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:31]: I might have to remove this. Waiting to hear back from PM if this means all users are able to see/use the **Geography** page.

File: src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx [hunk 2]
```diff
@@ -77,13 +76,13 @@ You can filter and group data by attributes in various categories; for example:
...
-  title="Filterable geography UI"
-  alt="Screenshot of Filterable geography UI in New Relic"
+  title="Geography UI"
+  alt="Screenshot of Geography UI in New Relic"
   src={browserGeoKeyAttributes}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos > Key attributes charts**: Here is an example of charts that have been filtered so you can examine network performance by ASN organizations.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Geography > Key attributes charts**: Here is an example of charts that have been filtered so you can examine network performance by ASN organizations.
```
Comment (akristen) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:85]:
```suggestion
  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser >** (select an app) > **Geography > Key attributes charts**: Here is an example of charts that have been filtered so you can examine network performance by ASN organizations.
```

Comment (akristen) [reply src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:85]: IIRC for actions not based in the UI, we break the bold. but i might x2 check in the style guide

**Review by akristen — DISMISSED** [review]
Looks good! Gonna approve; I'd x2 check the style guide for pathing styles. I like in your intro where you call out specific kinds of users and what utility those user types might get from the feature. Maybe you can differentiate that kind of info with bulleted lists somehow?

These are just suggestions and aren't make/break its!

File: src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx [hunk 3]
```diff
@@ -19,48 +20,46 @@ import browserFilterableGraphs from 'images/browser_screenshot-full_filterable-g
...
-  title="Filterable geography UI"
-  alt="Screenshot of Filterable geography UI in New Relic"
+  title="Geography UI"
+  alt="Screenshot of Geography UI in New Relic"
   src={browserGeoSummary}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos:** Get detailed information about network performance and end users' experience anywhere around the world.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Geography**: Get detailed information about network performance and end users' experience anywhere around the world.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:43]:
```suggestion
  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > **(select an app) > **Geography**: Get detailed information about network performance and end users' experience anywhere around the world.
```

File: src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx [hunk 4]
```diff
@@ -19,48 +20,46 @@ import browserFilterableGraphs from 'images/browser_screenshot-full_filterable-g
...
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos:** Get detailed information about network performance and end users' experience anywhere around the world.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > **(select an app) > **Geography**: Get detailed information about network performance and end users' experience anywhere around the world.
 </figcaption>
 
 To view or sort your browser monitoring performance data by location:
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos**.
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Geography**.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:48]:
```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > ** (select an app) **> Geography**.
```

File: src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx [hunk 5]
```diff
@@ -95,7 +94,7 @@ The filtered results appear on both the geographical map and the key attribute c
   >
     In this example, your organization is preparing to renew a peering agreement with the owner of an autonomous system, called `My-ASN`. You are concerned that continued outages and other connection issues are affecting customer satisfaction with your site, and you want to compare its performance with other ASNs.
 
-    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos**.
+    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Geography**.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:97]:
```suggestion
    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser >** (select an app) **> Geography**.
```

File: src/content/docs/browser/new-relic-browser/configuration/browser-settings-ui-options-apdex-geography.mdx [hunk 6]
```diff
@@ -17,15 +17,15 @@ As part of the browser [setup requirements](/docs/browser/new-relic-browser/welc
...
+* Cities, regions, or countries to be highlighted on the [**Geography** page](/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location)
 
 ## Select Apdex and geography settings [#apdex-settings]
 
-For apps that are also monitored by APM, you can set separate [Apdex T values](/docs/apm/new-relic-apm/apdex/apdex-measuring-user-satisfaction) for browser (end users) and for [APM](/docs/apm/new-relic-apm/apdex/changing-your-apdex-settings). You can also select cities, regions, or countries to be highlighted on the **Filterable geography** page.
+For apps that are also monitored by APM, you can set separate [Apdex T values](/docs/apm/new-relic-apm/apdex/apdex-measuring-user-satisfaction) for browser (end users) and for [APM](/docs/apm/new-relic-apm/apdex/changing-your-apdex-settings). You can also select cities, regions, or countries to be highlighted on the **Geography** page.
 
 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Settings > Application settings**.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/configuration/browser-settings-ui-options-apdex-geography.mdx:26]:
```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > **(select an app) **> Settings > Application settings**.
```

File: src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx [hunk 7]
```diff
@@ -19,48 +20,46 @@ import browserFilterableGraphs from 'images/browser_screenshot-full_filterable-g
...
-  title="Filterable geography UI"
-  alt="Screenshot of Filterable geography UI in New Relic"
+  title="Geography UI"
+  alt="Screenshot of Geography UI in New Relic"
   src={browserGeoSummary}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos:** Get detailed information about network performance and end users' experience anywhere around the world.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > **(select an app) > **Geography**: Get detailed information about network performance and end users' experience anywhere around the world.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:43]:
```suggestion
  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser >** (select an app) > **Geography**: Get detailed information about network performance and end users' experience anywhere around the world.
```

File: src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx [hunk 8]
```diff
@@ -107,7 +106,7 @@ The filtered results appear on both the geographical map and the key attribute c
   >
     In this example, your organization has a special three-day sales promotion for an upcoming holiday. Your `holiday-merchandise` page has undergone major changes, including dozens of new images. You want to look for performance problems with page rendering on Android mobile devices during this sales period.
 
-    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos**.
+    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Geography geos**.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:109]:
```suggestion
    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser >** (select an app) **> Geography geos**.
```

File: src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx [hunk 9]
```diff
@@ -19,48 +20,46 @@ import browserFilterableGraphs from 'images/browser_screenshot-full_filterable-g
...
-  title="Filterable geography charts UI"
-  alt="Screenshot of Filterable geography charts UI in New Relic"
+  title="Geography UI charts"
+  alt="Screenshot of Geography charts UI in New Relic"
   src={browserFilterableGraphs}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Filterable geos:** The charts on the **Map view** and **Key attribute charts** tabs provide additional details about page load and network performance over time.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Geography**: The charts on the **Map view** and **Key attribute charts** tabs provide additional details about page load and network performance over time.
```
Comment (a-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx:62]:
```suggestion
  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser >** (select an app) **> Geography**: The charts on the **Map view** and **Key attribute charts** tabs provide additional details about page load and network performance over time.
```

File: src/nav/browser.yml [hunk 10]
```diff
@@ -53,7 +53,7 @@ pages:
         path: /docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes
       - title: Session traces page
         path: /docs/browser/browser-monitoring/browser-pro-features/session-traces-explore-webpages-life-cycle
-      - title: Filterable geography page
+      - title: Geography page
         path: /docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location
```
Comment (a-sassman) [inline src/nav/browser.yml:57]:
```suggestion
        path: /docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location
```

**Pull request merged**